packages/react/src/index.d.ts

```diff
@@ -113,20 +300,6 @@ declare namespace React {
 		children: ReactNode;
 	}
 
-	//
-	// Factories
```

Comment:
Deletion is missing from previous  [PR](https://github.com/littensy/rbxts-react/pull/8).